config/config/wc-config.yaml

```diff
@@ -252,3 +255,18 @@ networkPolicies:
 
   alertmanager:
     enabled: true
+
+## Open policy agent configuration
+gatekeeper:
+  restrictUserCRDs:
+    enabled: false
+    enforcement: deny
+
+  sealedSecrets:
+    enabled: false
+
+  mongodb:
+    enabled: false
+
+  userCRDs:
+    allowedCRDs: []
```

Comment:
Could you add an example as a comment here? So it's clear what is needed if you want to add a CRD to the list.